R/source_utils.R

```diff
@@ -0,0 +1,21 @@
+#' Build the `xml_find_function_calls()` helper for a source expression
+#'
+#' @param xml The XML parse tree as an XML object (`xml_parsed_content` or `full_xml_parsed_content`)
+#'
+#' @return A fast function to query
+#' `xml_find_all(xml, glue::glue("//SYMBOL_FUNCTION_CALL[{ xp_text_in_table(function_names) }]"))`.
```

Comment:
NB: `xp_text_in_table()` isn't (yet?) exported. Not important b/c this is `@noRd` right?

Reply:
Added another example without `xp_text_in_table()` just to be sure.